### DISCUSSION TITLE
Input voltage and resetting Faikin

### DISCUSSION THREAD

**Discussion by tiekenz** [revk/xESP32-Faikout#67] [Q&A] [answered]
As the module doesn't work with my system, I was going to pass it off to someone else, but realised I had set it to my wifi and I don't have access to my heatpump at the moment to re-power it.
I thought I should just provide it with local power and reset it, but thought I should check first - does it run at the standard ESP32 voltage? (ie 2.5V>3.6V?) and is there a standard reset method?
Thanks.

**Comment by revk** [comment, marked as answer]
It can handle from 4V to 36V on the DC pin.
